app/actions/email.ts

```diff
@@ -0,0 +1,69 @@
+import { SEND_MAIL } from "../constants/actionTypes";
+import { MAIN } from "../constants/targets";
+import { generateInPlaceDescription } from "../flat/applicationTextBuilder";
+import { flatPageUrl } from "../flat/urlBuilder";
+import type { ContactData } from "../reducers/configuration";
+import type { OverviewDataEntry } from "../reducers/data";
+import type { Action, Dispatch, GetState } from "../reducers/types";
+import { markCompleted } from "./cache";
+import { CACHE_NAMES } from "../reducers/cache";
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
pretty sure this `esline-disable-next-line` is unnecessary, could you check if we have "no unnecessary rule disabling" active and if yes, why this didn't yield an error?

Reply:
removed the disable

could not find the "no unnecessary disabling" thingy...

Reply:
https://github.com/mysticatea/eslint-plugin-eslint-comments/blob/master/docs/rules/no-unused-disable.md
Not sure how easy it is to set up and if it's worth it, could you look into that?